make-docs.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# Execute from the repo's root directory
+rm -rf docs/source/*.txt
+SPHINX_APIDOC_OPTIONS=members,undoc-members,show-inheritance sphinx-apidoc -eM -s txt -o docs/source/ exa *test*
+cd docs
```

Comment:
If we use it in the travis build don't we have to cd to the home dir?

Reply:
The `travis-sphinx build` command publishes to a specific branch containing the docs only (gh pages). I don't think it makes sense to use locally.

Reply:
Yes, but I think this comment is outdated with what is said above for the .travis.yml.